src/components/EventCard/EventCard.jsx

```diff
@@ -4,7 +4,7 @@ import Detail from '../Detail/Detail';
 
 import EventCardContainer from './EventCard.styled';
 
-const EventCard = ({event}) => {
+const EventCard = ({event, deleteEvent}) => {
 	const [details, setDetails] = useState(false);
 
 	const switcher = () => {
```

Comment:
Maybe you can think of a more specific name for your variable, but this is a really minor comment.

Reply:
done, thanks for the suggestion